Gruntfile.js

```diff
@@ -165,6 +165,21 @@ module.exports = function (grunt) {
                 src: 'src/main/webapp/js/min/script.js',
                 dest: 'src/main/webapp/js/min/script.clean.js'
             }
+        },
+        protractor : {
+            options : {
+                keepAlive : true,
```

Comment:
When 'keepAlive : true' maven task (grunt protractor) never fails irrespective of the protractor results